permissions/resource_permissions.go

```diff
@@ -245,15 +245,6 @@ func (a PermissionsAPI) Delete(objectID string) error {
 			UserName:        job.CreatorUserName,
 			PermissionLevel: "IS_OWNER",
 		})
-	} else if strings.HasPrefix(objectID, "/sql/warehouses") {
```

Comment:
Can this be a regression if we are not setting owner automatically?

Reply:
We are prioritising https://github.com/databricks/terraform-provider-databricks/issues/2678 since many people are blocked on this. There might be an issue with DBSQL API that we need to followup with. I have verified that setting IS_OWNER works for a username so this would only affect people that want to have owner set automatically.

Even in that case, people won't be unblocked -- they can set the permission in TF explicitly. 

So going ahead with this change would be good.